build/Dockerfile

```diff
@@ -34,16 +33,16 @@ RUN python build.py
 
 RUN /work/build/ubuntu-latest/kubescape download artifacts -o /work/artifacts
 
-FROM alpine
+FROM alpine:3.16.2
 
 RUN addgroup -S armo && adduser -S armo -G armo
 
-RUN mkdir /home/armo/.kubescape
```

Comment:
Why did you remove this line?

Reply:
COPY command creates the destination if one doesn't exist.

https://docs.docker.com/engine/reference/builder/#copy